src/ast/ast_validations.c

```diff
@@ -1509,6 +1510,70 @@ static AST_Validation _BlockUnsupportedMerges(AST *ast, char **reason) {
 	return res;
 }
 
+static void _colect_query_parameters_names(const cypher_astnode_t *root, rax *keys) {
+	cypher_astnode_type_t type = cypher_astnode_type(root);
+	// In case of identifier.
```

Comment:
Fix comment - parameter, not identifier.

Reply:
fixed